### PR TITLE
Draft of json objects for patient Z058.

### PR DESCRIPTION
**Pull request by sglanger**
Only had text for 1 CT, 1 PET/CT and path

### CODE REVIEW

File: TCGA-17-Z058/Patient/patient.z058.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            "assigner": {
+                "display": "Acme Healthcare"
+            }
+        }
+    ],
+    "name": [
+        {
+            "use": "official",
+            "family": [
+                "TCGA-17-Z058
```
Comment (shadowdoc) [inline TCGA-17-Z058/Patient/patient.z058.json]: Missing a closing quote on this line.

File: TCGA-17-Z058/Patient/patient.z058.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
+// patient.example.json
+// id: z058
```
Comment (shadowdoc) [inline TCGA-17-Z058/Patient/patient.z058.json]: I think that these are case sensitive and you use Z058 elsewhere

File: TCGA-17-Z058/Specimen/specimen.z058.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        "coding": [
+            {
+                "system": "http://snomed.info/sct",
+                "code": "122555007",
+                "display": "Lingular lung bxs"
+            }
+        ]
+    },
+    "subject": {
+        "reference": "Patient/example",
```
Comment (shadowdoc) [inline TCGA-17-Z058/Specimen/specimen.z058.json]: should be: "reference":  "Patient/Z058",

File: TCGA-17-Z058/Condition/condition.z058.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+// http://fhir.hackathon.siim.org/fhir/Condition/example/?_format=application/json
+
+{
+    "resourceType": "Condition",
+    "text": {
+        "status": "generated",
+        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">60 yo male with history of lung adenocarcinoma (Date: 24-May 1985)</div>"
+    },
+    "subject": {
+        "reference": "Patient/example"
```
Comment (shadowdoc) [inline TCGA-17-Z058/Condition/condition.z058.json]: should be: "reference": "Patient/Z058",

**Comment by sglanger** [comment]
made corrections, try now

File: TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
+// diagnostic_report.Z058.ct0433.json
+// id: Z058
```
Comment (shadowdoc) [inline TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json]: this ID should be Z058ct0443

File: TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.path.json [hunk 6]
```diff
@@ -0,0 +1,71 @@
+// diagnostic_report.Z058.path.json
+// id: Z058
```
Comment (shadowdoc) [inline TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.path.json]: this should be Z058path

File: TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.pt5759.json [hunk 7]
```diff
@@ -0,0 +1,71 @@
+// diagnostic_report.Z058.pt5759.json
+// id: Z058
```
Comment (shadowdoc) [inline TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.pt5759.json]: Z058pt5759

File: TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json (hunk 5 above)
Comment (sglanger) [inline TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json]: I have made it a patter to isolate the patient "name" from the "exam" name

The Breast Dx reports follow the same pattern



________________________________

Comment (shadowdoc) [inline TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json]: I agree regarding the file names, but the IDs cannot have punctuation inside them.  Check out the IDs for the BreastDx patients.

Comment (sglanger) [inline TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json]: Ah, of course. Try now


________________________________

**Pull request merged**